### DISCUSSION TITLE
Router Flash For Sonoff Zig 3.0 USB Dongle Plus

### DISCUSSION THREAD

**Discussion by Shaunpottie** [Koenkk/Z-Stack-firmware#427] [Q&A]
Hey everyone.

Please can you help?

I bought a second Sonoff Zigbee Dongle Plus, to setup as a router, tried flashing it but it just keeps saying >Unknown record type: 3.

Yet when I flash it with the Coordinator software, it flashes fine no problems?

My method is flashing via USB with Flash Programmer 2

Thank you in advance

Regards,
Shaun

**Comment by grannysmith59** [comment]
I thought I had posted this, I experienced exactly the same last night!

I bought a pack of two, and had the same issue with both.

**Comment by rwunsch** [comment]
Could you please check :
https://github.com/Koenkk/Z-Stack-firmware/issues/423

**Comment by rwunsch** [comment]
and/or:
https://github.com/Koenkk/Z-Stack-firmware/issues/397

**Comment by Shaunpottie** [comment]
Thank you, came right with Zig Star software, worked with a little fiddling.

Followed this post here https://github.com/Koenkk/Z-Stack-firmware/issues/397
